### ISSUE TITLE
Disable/pause webcam track/source

### ISSUE THREAD

**Issue by efsjouw** [microsoft/MixedReality-WebRTC#665]
Hello, is it possible to disable/pause the webcam source or track while still recording the scene.

**Comment by djee-ms** [comment]
> is it possible to disable/pause the webcam source...

No, by (Google's) design a source is always active until it's destroyed.

> ...or track...

Yes. Just mute (== disable) the track. Or remove it from its transceiver (replace it with a NULL track). Note that the webcam will continue to record, but the video frames will be discarded, so this is fine for a short pause but not ideal (CPU, thermal, battery) for longer interruptions.
